src/_data/styles/fresh.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Fresh",
+  "author": "Thomas Champion",
+  "stylesheet": "https://raw.githubusercontent.com/Tombiliby/stylestage-sass/main/public/style.css",
+  "websiteUrl": "http://thomaschampion.info/"
```

Comment:
Style updates look good - just need to add `websiteTitle` so that the link has clickable text, such as `"websiteTitle": "ThomasChampion.info"`